test/e2e_node/memory_manager_test.go

```diff
@@ -230,12 +231,16 @@ func getUpdatedKubeletConfig(oldCfg *kubeletconfig.KubeletConfiguration, params
 	return newCfg
 }
 
-func updateKubeletConfig(f *framework.Framework, cfg *kubeletconfig.KubeletConfiguration) {
-	// remove the state file
+func updateKubeletConfig(f *framework.Framework, cfg *kubeletconfigv1beta1.KubeletConfiguration) {
+	ginkgo.By("Stopping the kubelet")
+	stopKubelet()
```

Comment:
I think all tests restarting kubelet needs to be marked as `[Disruptive]`. This will prevent picking it up by mistake by some tabs

Reply:
Good idea! Done